scripts/methods.rb

```diff
@@ -33,6 +33,12 @@ def parse_sheets
       data_to_insert = [HEADINGS_INSERT, map_data(row, headings)]
       begin
         x = data_to_insert.transpose.to_h
+        x.each do |key, value|
+          if value != nil and value.is_a? Spreadsheet::Excel::Error
+            puts "ERROR: #{file} #{x['IFSC']}"
```

Comment:
@captn3m0 Will drop the nil check. I put the `puts` there so that the user gets a general idea of the number of errors that might be there in the Excel sheets. Do you still think it should be removed?